trashcli/put.py

```diff
@@ -48,25 +58,19 @@ def __init__(self, stdout, stderr, environ, trasher):
     def run(self, argv):
         program_name = os.path.basename(argv[0])
         parser = get_option_parser(program_name, self.stdout, self.stderr)
-        try:
-            (options, args) = parser.parse_args(argv[1:])
+        options = parser.parse_args(argv[1:])
 
-            if len(args) <= 0:
-                parser.error("Please specify the files to trash.")
```

Comment:
Please restore the old message (`"Please specify the files to trash."`). You may need to change the value of nargs in the lines below
```
    parser.add_argument('files',
                        nargs='+')
```

Reply:
Fixed.

Reply:
Good 👍